extensions/wikia/AdEngine/js/lookup/adapter/appnexusPlacements.js

```diff
@@ -0,0 +1,32 @@
+define('ext.wikia.adEngine.lookup.adapter.appnexusPlacements', [
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.window'
+], function (zoneParams, win) {
+	'use strict';
+
+	var placements = {
+		mercury: {
+			entertainment: isDevEnviroment() ? '9412980' : '9412992',
```

Comment:
you think it's better to end up with 12 right now (and more in the future) constants? For me this way it's more readable because you don't need to jump to some variable to know the value. This number won't be reused anywhere, it's per skin-domain-vertical combination. Possibly in the future per slot as well.